### PR TITLE
added distanceToSelf output to "Traffic" section

### PR DESCRIPTION
**Pull request by humppafreak**

### CODE REVIEW

File: calcs/cpa_tcpa.js [hunk 1]
```diff
@@ -121,6 +127,38 @@ module.exports = function (app, plugin) {
...
+              context: 'vessels.' + vessel,
+              updates: [
+                {
+                  values: [
+                    {
+                      path: 'distanceToSelf',
+                      value: distance
+                    }
+                  ],
+                  meta: [
```
Comment (tkurki) [inline calcs/cpa_tcpa.js:146]: Sending meta on each update bloats them needlessly. Once should suffice - the server will make the meta available for downstream apps.

File: calcs/cpa_tcpa.js [hunk 2]
```diff
@@ -121,6 +127,38 @@ module.exports = function (app, plugin) {
...
+            plugin.properties.traffic.distanceToSelf
+         ) {
+            app.debug('distance of ' + vessel + ' to self: ' + distance)
+            app.handleMessage(plugin.id, {
+              context: 'vessels.' + vessel,
+              updates: [
+                {
+                  values: [
+                    {
+                      path: 'distanceToSelf',
```
Comment (tkurki) [inline calcs/cpa_tcpa.js:142]: Maybe `navigation.distanceToSelf`?

**Review by tkurki — CHANGES_REQUESTED** [review]

**Comment by tkurki** [comment]
Still on this?

File: calcs/cpa_tcpa.js (hunk 1 above)
Comment (humppafreak) [reply calcs/cpa_tcpa.js]: removed line 132 (enabled calc by default) and deleted meta from update.

**Pull request merged**